tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf

```diff
@@ -0,0 +1,58 @@
+# Get worker security group.
+data "aws_security_group" "worker_sg" {
+  vpc_id = data.aws_vpc.env_vpc.id
+
+  filter {
+    name   = "tag:kubernetes.io/cluster/${var.cluster_id}"
+    values = ["owned"]
+  }
+
+  filter {
+    name   = "tag:Name"
+    values = ["${var.cluster_id}-worker-sg"]
+  }
+}
+
+# Add a rule for vxlan traffic for all workers.
+resource "aws_security_group_rule" "worker_sg_vxlan_rule" {
+  security_group_id        = data.aws_security_group.worker_sg.id
+  source_security_group_id = data.aws_security_group.worker_sg.id
+  from_port                = 4800
+  protocol                 = "udp"
+  to_port                  = 4800
+  type                     = "ingress"
+}
+
+
+# Create a submariner agteway security group.
+resource "aws_security_group" "submariner_gw_sg" {
+  name   = "${var.cluster_id}-submariner-gw-sg"
+  vpc_id = data.aws_vpc.env_vpc.id
+
+  egress {
+    from_port = 0
+    protocol  = "-1"
+    to_port   = 0
+
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```

Comment:
Why do we require this SG rule? Is egress not allowed by default?

Reply:
Not sure, I guess probably not, I'm not the author and don't have capacity to check it at this point, but I can add a note to verify in the future. Sounds ok?

Reply:
Sure

Reply:
Done

Reply:
We can remove the egress part, there is an additional egress rule on the the second security group created by the installer.

Reply:
ack, I will remove that part now.